docs/HowToFAQ/FAQSplitHere.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
+---
+
+## Question
+
+How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
+
+## Answer
+
+Analysis of a method for the purpose of verification identifies a large number of assertions that must be proved.
+These assertions are grouped into batches.
```

Comment:
I think it might be good to lead with the reason for grouping assertions into batches.

`Dafny uses an SMT solver to verify assertions. The SMT solver may become better or worse at verifying an assertion if you also ask it to also verify another assertion. Dafny allows you to split up a group of assertions into batches, where each batch is sent to the SMT solver separately, so the verification of each batch is not influenced by the others.`

Reply:
Reworked to include this text